src/mslice/scripting/helperfunctions.py

```diff
@@ -107,23 +107,23 @@ def add_overplot_statements(script_lines, plot_handler):
         recoil = True if rmm is not None or key in [1, 2, 4] else False
         cif = None  # Does not yet account for CIF files
 
+        ws_var = ws_vars.pop(0) if ws_vars else f"'{plot_handler.ws_name}'"
```

Comment:
Does the pop bit need single quotes around it?

```suggestion
        ws_var = f"'{ws_vars.pop(0)}'" if ws_vars else f"'{plot_handler.ws_name}'"
```

Reply:
I don't _think_ so, as ws_vars.pop(0) returns a string as it is.

Reply:
Yes I think it does return a string, however when it is input into the python code in the `script_lines` variable, it will not have quotes around the variable name and so it doesn't think its a string inside the python script we create. e.g. the color string variable needs quotes
```
ax.recoil(workspace={ws_var}, rmm={rmm}, color='{color}')\n
```

Reply:
Ah I see what you mean now, this is intentional in this case, the `ws_var` refers to a python variable that we assign to earlier in the script, not the name of a workspace in the form of a string.